demo/play-demo-for-recording.sh

```diff
@@ -0,0 +1,208 @@
+    <packaging>pom</packaging>
+    <modules>
+        <module>fruit-backend-sb</module>
+        <module>fruit-client-sb</module>
+    </modules>
+</project>
+ENDOFFILE
+clear && sleep 1
+echo "# Create a client - microservice using the REST HTTP client template proposed by the scaffolding tool"|pv -qL 10
+sleep 3
+exec hal component spring-boot \
+   -i fruit-client-sb \
+   -g me.fruitstand \
+   -p me.fruitstand.demo \
+   -s 2.1.6.RELEASE \
+   -t client \
+   -v 1.0.0-SNAPSHOT \
+   --supported=false  \
+   fruit-client-sb
+sleep 3
+
+clear && sleep 1
+echo "# Create a backend - microservice interactively using now the CRUD template"|pv -qL 10
+sleep 3
+exec hal component spring-boot fruit-backend-sb
+sleep 3
+
+clear && sleep 1
+echo "# Build the Spring Boot microservices"|pv -qL 10
+echo "# Compile and generate the Spring Boot uber jar file for the REST Client microservice"|pv -qL 10
+sleep 3
+exec mvn package -f fruit-client-sb
+sleep 3
+
+clear && sleep 1
+echo "# Repeat the command executed previously for the CRUD - backend microservice."|pv -qL 10
+echo "# We need to use the kubernetes profile because the project is set up to work both locally using H2 database for quick testing and \"remotely\" using a PostgreSQL database."|pv -qL 10
+sleep 4
+exec mvn package -f fruit-backend-sb -Pkubernetes
+sleep 3
+
+clear && sleep 1
+echo "# Create for each microservice a component and deploy it on the cluster"|pv -qL 10
+sleep 2
+exec hal component create -c fruit-client-sb
+sleep 2
+exec hal component create -c fruit-backend-sb
+sleep 3
+
+clear && sleep 1
+echo "# Check that the components have been correctly installed"|pv -qL 10
+sleep 2
+exec oc get cp
+sleep 3
+
+clear && sleep 1
+echo "# Create a capability to install a PostgreSQL database using the interactive mode"|pv -qL 10
+sleep 2
+exec hal capability create -n postgres-db
+sleep 3
+
+clear && sleep 1
+echo "# Check the capability status"|pv -qL 10
+sleep 2
+exec oc get capabilities
+sleep 10
+
+clear && sleep 1
+echo "# Link the microservices"|pv -qL 10
+echo "# We need to wire the fruit-backend-sb component with the postgres-db capability by creating a link between them"|pv -qL 10
+sleep 4
+exec hal link create -n backend-to-db -t fruit-backend-sb
+sleep 3
+
+clear && sleep 1
+echo "# Now, create a link targeting the fruit-client-sb component to wire the client and the backend microservices"|pv -qL 10
+sleep 3
+exec hal link create -n client-to-backend -t fruit-client-sb -e KUBERNETES_ENDPOINT_FRUIT=http://fruit-backend-sb:8080/api/fruits
+sleep 3
+
+clear && sleep 1
+echo "# Check the link status"|pv -qL 10
+sleep 2
+exec oc get links
+sleep 3
+
+clear && sleep 1
+echo "# Push your local code source (e.g pom.xml, ./src,...) to the pod"|pv -qL 10
+sleep 2
+exec hal component push -c fruit-client-sb,fruit-backend-sb
+echo "# Let's wait to let maven to build the code within the pod and next to start the Spring Boot application"|pv -qL 10
```

Comment:
`Let's wait a few seconds to let maven build the Spring Boot application on the cluster. Halkyon will then automatically start the app`.